pkg/mimirpb/timeseries.go

```diff
@@ -309,7 +315,9 @@ func (p *PreallocTimeseries) MarshalToSizedBuffer(buf []byte) (int, error) {
 }
 
 // LabelAdapter is a labels.Label that can be marshalled to/from protos.
-type LabelAdapter labels.Label
+type LabelAdapter struct {
```

Comment:
`LabelAdapter` is reused in too many places (and, for what I could find, safely so) to just rename it to `UnsafeMutableLabel`, hence the alias.